pytorch_lightning/strategies/horovod.py

```diff
@@ -112,6 +118,12 @@ def _unpack_lightning_optimizer(opt):
         for optimizer in optimizers:
             hvd.broadcast_optimizer_state(optimizer, root_rank=0)
 
+        accumulation_scheduler = self.lightning_module.trainer.accumulation_scheduler
```

Comment:
trainer is an argument to this function already
```suggestion
        accumulation_scheduler = trainer.accumulation_scheduler
```